scripts/ci/version_mgr.py

```diff
@@ -2,67 +2,126 @@
 # Copyright (c) 2020 Intel Corp.
 # SPDX-License-Identifier: Apache-2.0
 
+"""
+Syntax of file:
+    [
+        {
+            "version": "<commit>",
+            "date": "<date>",
+            "weekly: False,
+        },
+    ]
+"""
 import json
 import argparse
 import urllib.request
+import os
 
 from git import Git
+from datetime import datetime
 
 VERSIONS_FILE = "versions.json"
 
+
 def parse_args():
     parser = argparse.ArgumentParser(
                 description="Manage versions to be tested.")
     parser.add_argument('-l', '--list', action="store_true",
-            help="List all published versions")
+                        help="List all published versions")
     parser.add_argument('-u', '--update',
-            help="Update versions file from tree.")
+                        help="Update versions file from tree.")
     parser.add_argument('-L', '--latest', action="store_true",
-            help="Get latest published version")
+                        help="Get latest published version")
+    parser.add_argument('-w', '--weekly', action="store_true",
+                        help="Mark as weekly")
```

Comment:
Could you add that a default option is "true" in the help?

Reply:
I think better to add Readme with usage examples

Reply:
I can make a PR with usage examples, after that PR get merged.

Reply:
just to be clear, you are not supposed to use the -w command, this is just for CI

Reply:
> I found out that if you run command above again and again, the same version will be added many times. @nashif Is there necessary to add a check if version is already in the file, or that case won't happen during usage of that script?

there is a check already, but there was a bug, fixed now.